.github/workflows/rotki_nightly.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Download Geth
         run: |
           .travis/download_geth.sh ${{ matrix.os }}
-          echo "::add-path::$HOME/.bin"
+          echo "PATH=$HOME/.bin:$PATH" >> $GITHUB_ENV
```

Comment:
The `add-path` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/